core/domain/classifier_services.py

```diff
@@ -305,9 +313,9 @@ def _update_classifier_training_job(classifier_training_job_model, status):
     classifier_training_job_model.put()
 
 
-def save_classifier_training_job(algorithm_id, exp_id, exp_version,
-                                 state_name, status, training_data,
-                                 job_id="None"):
+def save_classifier_training_job(algorithm_id, exp_id,
```

Comment:
I've read this file a couple times, and I'm a bit confused about how we're factoring this. Thinking about it from the perspective of an external client file that wants to use the stuff in this module, wouldn't we want:

- create_new_training_job (called when exploration is saved)
- mark_training_job_complete (called when data comes back from the VM)
- mark_training_job_pending (called when job request is sent to the VM)

and all of these (and possibly others) call an internal _update_classifier_training_job function? It seems to me that the modelling is a bit backwards here...